lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua

```diff
@@ -0,0 +1,126 @@
+CFCE2Lib.RegisterExtension( "cfc_e2_lib", true )
+
+-- Made by StrawWagen
+
+-- Setting up local functions 
+
+local function isValidPly( ply )
+    if not ply or not ply:IsValid() or not ply:IsPlayer() then
+        return false
+    end
+    return true
+end
+
+local function isValidNpc( npc )
+    if not npc or not npc:IsValid() or not npc:IsNPC() then
+        return false
+    end
+    return true
+end
```

Comment:
I don't think either of these are necessary (maybe the NPC check), but if you're going to keep them, they should be refactored to follow our style guidelines
```suggestion
local function isValidPly( ply )
    if not IsValid( ply ) then return false end
    if not ply:IsPlayer() then return false end
    
    return true
end

local function isValidNpc( npc )
    if not IsValid( npc ) then return false end
    if not npc:IsNPC() then return false end
    
    return true
end
```